File: src/main/java/mods/betterfoliage/client/integration/EFRIntegration.java
package mods.betterfoliage.client.integration;

import net.minecraft.block.Block;

import ganymedes01.etfuturum.ModBlocks;
import ganymedes01.etfuturum.blocks.BlockModernLeaves;

public final class EFRIntegration {

    // TODO: ported to Java, but might not be needed after configs refactor
    public static boolean isEFRCherryLeaves(Block block, int meta) {
        return Mod.EFR.isLoaded() && block instanceof BlockModernLeaves && meta % 4 == 1;
    }

    public static boolean isEFRSoulSoil(Block block) {
        return Mod.EFR.isLoaded() && ModBlocks.SOUL_SOIL.isEnabled() && block == ModBlocks.SOUL_SOIL.get();
    }
}

File: src/main/kotlin/mods/betterfoliage/client/integration/EFRIntegration.kt
package mods.betterfoliage.client.integration

import cpw.mods.fml.relauncher.Side
import cpw.mods.fml.relauncher.SideOnly
import ganymedes01.etfuturum.blocks.BlockModernLeaves
import net.minecraft.block.Block

@SideOnly(Side.CLIENT)
object EFRIntegration {
    @JvmStatic fun isETFCherryLeaves(block: Block, meta: Int) = Mod.EFR.isLoaded && block is BlockModernLeaves && meta % 4 == 1
}
